README.md

```diff
@@ -1,6 +1,25 @@
-# hedera-block-node
-The block node is a new and unique node designed to consume the blocks streams, maintain state and expose additional targeted value adding APIs to the Hedera community.
-More details to come as this is work in progress
+[![Build Application](https://github.com/hashgraph/hedera-block-node/actions/workflows/build-application.yaml/badge.svg?branch=main)](https://github.com/hashgraph/hedera-block-node/actions/workflows/build-application.yaml)
+[![E2E Test Suites](https://github.com/hashgraph/hedera-block-node/actions/workflows/e2e-tests.yaml/badge.svg?branch=main)](https://github.com/hashgraph/hedera-block-node/actions/workflows/e2e-tests.yaml)
+[![codecov](https://codecov.io/github/hashgraph/hedera-block-node/graph/badge.svg?token=OF6T6E8V7U)](https://codecov.io/github/hashgraph/hedera-block-node)
+
+[![Latest Version](https://img.shields.io/github/v/tag/hashgraph/hedera-block-node?sort=semver&label=version)](README.md)
+[![Made With](https://img.shields.io/badge/made_with-java-blue)](https://github.com/hashgraph/hedera-block-node/)
+[![Development Branch](https://img.shields.io/badge/docs-quickstart-green.svg)](docs/gradle-quickstart.md)
+[![License](https://img.shields.io/badge/license-apache2-blue.svg)](LICENSE)
+
+# Hedera Block Node
+Implementation of the Hedera Block Node, which is responsible for consuming the block streams, maintaining state and exposing additional targeted value adding APIs to the Hedera community.
+
+## Overview of child modules
+
+- `server`: implementation of the block node, which contains the main application and all the necessary code to run the block node.
+- `simulator`: A simulator for the block node, which can be used to test the block node in a local environment.
+- `common`: Module responsible for holding common literals, utilities and types used by the other modules.
+- `suites`: A set of e2e tests that can be used to verify the correctness of the block node.
```

Comment:
No, this is responsibility of the owner of this module, or the one that introduced it. If it get's merged before this PR, I will add it, otherwise it should be added in the PR that introduces it.

Reply:
Agreed for the responsibility, just a friendly reminder then not to forget for the docs to be added (by the owner of the respective module), so if this goes in first, we could ping the owner to add the docs, otherwise we could make an issue to be added docs so the owner can follow up.